src/gamemode/gametypemenu/menu.asm

```diff
@@ -101,6 +101,7 @@ gameTypeLoopCheckStart:
         lda set_seed_input
```

Comment:
can you update the comment on line 98 so we remember what the and is for =)

Reply:
something like:

```
and #$FE ; least significant bit is insignificant, 0001 should be treated like 0000.
```
?

Reply:
yeah, just needs some thing so that someone reading it knows why it's there